src/navigation_controller.ts

```diff
@@ -552,6 +552,34 @@ export class NavigationController {
       keyCodes: [KeyCodes.ENTER],
     },
 
+    /**
+     * Cmd/Ctrl/Alt+Enter key:
+     *
+     * Shows the action menu.
+     */
+    menu: {
+      name: Constants.SHORTCUT_NAMES.MENU,
+      preconditionFn: (workspace) => this.canCurrentlyNavigate(workspace),
+      callback: (workspace) => {
+        switch (this.navigation.getState(workspace)) {
+          case Constants.STATE.WORKSPACE: {
+            const node = workspace.getCursor()?.getCurNode();
+            if (node?.getType() === Blockly.ASTNode.types.BLOCK)
+              this.navigation.openActionMenu(node);
+
+            return true;
+          }
+          default:
+            return false;
+        }
+      },
+      keyCodes: [
+        createSerializedKey(KeyCodes.ENTER, [KeyCodes.CTRL]),
```

Comment:
I've followed the pattern for cut/copy/paste here, but alternatively we could check the platform at this point. Or the shortcut mechanism could have a way of defining this kind of platform specific shortcut. Would any platform use Alt in this way?

Reply:
Noted in https://github.com/google/blockly-keyboard-experimentation/issues/155#issuecomment-2643431314 to centralize responsibility for choosing the appropriate meta key.

In my (windows + linux) experience, Alt operates on windows.

Reply:
I'm not convinced Alt+C and similar on Windows are real. Raised https://github.com/google/blockly-keyboard-experimentation/issues/258